src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.NuGetOfflineCache.targets

```
@@ -14,12 +14,16 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <_NugetFallbackFolder>$(MSBuildThisFileDirectory)..\..\..\..\NuGetFallbackFolder</_NugetFallbackFolder>
     <_IsNETCore1x Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' &lt; '2.0' ">true</_IsNETCore1x>
+
+    <_WorkloadLibraryPacksFolder>$(MSBuildThisFileDirectory)..\..\..\..\..\library-packs</_WorkloadLibraryPacksFolder>
```

I'd suggest the following.  It follows the pattern of `NetCoreTargetingPackRoot` from the bundled versions file.  It will end up being the normalized path, and allows you to set the path to something else if you want to.

If you do this, you should re-test to make sure NuGet is OK with the trailing slash.

```suggestion
    <_WorkloadLibraryPacksFolder Condition="'$(_WorkloadLibraryPacksFolder)' == ''">$([MSBuild]::EnsureTrailingSlash('$(NetCoreRoot)'))library-packs</_WorkloadLibraryPacksFolder>
```

---

Tested. It works